itest/rust/src/object_tests/dyn_gd_test.rs

```diff
@@ -56,25 +56,48 @@ fn dyn_gd_creation_deref() {
 
 #[itest]
 fn dyn_gd_creation_deref_multiple_traits() {
-    let obj = foreign::NodeHealth::new_alloc();
-    let original_id = obj.instance_id();
+    let original_obj = foreign::NodeHealth::new_alloc();
+    let original_id = original_obj.instance_id();
 
-    // `dyn Health` must be explicitly declared if multiple AsDyn<...> trait implementations exist.
-    let mut obj = obj.into_dyn::<dyn Health>();
+    // Type can be inferred because `Health` explicitly declares
+    // 'static as its supertrait.
+    let mut obj = original_obj.clone().into_dyn();
```

Comment:
```suggestion
    // Type can be inferred because `Health` explicitly declares a 'static bound.
    let mut obj = original_obj.clone().into_dyn();
```
Line width 140-150 (see code style)

Also, `'static` isn't a trait, so it's not a supertrait either.